docusaurus/docs/resources/native-form.md

```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
 
 ### Required params
 
--   spaceId: String
+- spaceId: String
 
 ### Optional params
 
--   params: Object. Additional parameters you want sent in the post.
--   formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
--   type: String. Override the magneto integration type. Defaults to "saml"
+- params: Object. Additional parameters you want sent in the post.
+- formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
+- type: String. Override the magneto integration type. Defaults to "saml"
```

Comment:
As of 4.0.0 this no longer defaults to `saml`, I missed these docs in my PR.

Reply:
There's also one more param now for `clientId` which is optional and defaults to `'clientId'` if not provided.